core/src/main/java/org/fao/geonet/util/XslUtil.java

```diff
@@ -628,4 +634,61 @@ public static String encodeForURL(String str) {
             return str;
         }
     }
+
+	/**
+	 * @param dateFormat	Date format part supported by the SimpleDateFormat java class
+	 * @return	Formatted current date
+	 */
+    public static String getCurrentDate(String dateFormat) {
+		return new SimpleDateFormat(dateFormat).format(new Date());
```

Comment:
Method added but not used in this PR. Is this useful ? BTW, XSL provides similar capability eg. https://github.com/geonetwork/core-geonetwork/blob/develop/web/src/main/webapp/xsl/conversion/OGCWxSGetCapabilitiesto19119/OGCWxSGetCapabilities-to-19119.xsl#L137